extensions/wikia/AdEngine/js/provider/gpt/googleTag.js

```diff
@@ -1,14 +1,15 @@
 /*global define, require*/
 /*jshint maxlen:125, camelcase:false, maxdepth:7*/
 define('ext.wikia.adEngine.provider.gpt.googleTag', [
+	'ext.wikia.adEngine.provider.gpt.adSizeConverter',
```

Comment:
Shouldn't it be `adSizeMap`? Anyway this dependency seems not needed

Reply:
aaa, ofc, it's not needed.